public/app/features/auth-config/AuthConfigPage.tsx

```diff
@@ -115,6 +115,9 @@ export const AuthConfigPageUnconnected = ({ providerStatuses, isLoading, loadSet
                 authType={provider.protocol}
                 enabled={providerStatuses[provider.id]?.enabled}
                 configPath={provider.configPath}
+                onClick={() => {
```

Comment:
This is simpler form
```suggestion
                onClick={() => onProviderCardClick(provider)}
```